docs/stcal/ramp_fitting/description.rst

```diff
@@ -317,3 +320,39 @@ that pixel will be flagged as JUMP_DET in the corresponding integration in the
 "rateints" product.  That pixel will also be flagged as JUMP_DET in the "rate"
 product.
 
+Likelihood Algorithm Details
+----------------------------
+As an alternative to the OLS algorithm, a likelihood algorithm can be selected
+with for ``--ramp_fitting.algorithm=LIKELY``.  If this algorithm is selected,
+the normal pipeline jump detection algorithm is skipped because this algorithm
+has its own jump detection algorithm.  The jump detection for this algorithm
+requires NGROUPS to be a minimum of four (4).  If NGROUPS :math:`\le` 3, then
+this algorithm is deselected, defaulting to the above described OLS algorithm
+and the normal jump detection pipeline step is run.
+
+Each pixel is independently processed, but rather than operate on the each
+group/resultant directly, the likelihood algorithm is based on differences of
+the groups/resultants :math:`d_i = r_i - r_{i-1}`.  The model used to determine
+the slope/countrate, :math:`a`, is:
+
+.. math::    
+    \chi^2 = ({\bf d} - a \cdot {\bf 1})^T C ({\bf d} - a \cdot {\bf 1}) \,,
+
+Differentiating, setting to zero, then solving for :math:`a` results in 
+
+.. math::    
+    a = ({\bf 1}^T C {\bf d})({\bf 1}^T C {\bf 1})^T \,,
+
+The covariance matrix :math:`C` is a tridiagonal matrix, due to the nature of the
+differences.  Because the covariance matrix is tridiagonal, the  computational
+complexity from :math:`O(n^3)` to :math:`O(n)`.  To see the detailed derivation
+and computations implemented, refer to 
+`Brandt (2024) <https://iopscience.iop.org/article/10.1088/1538-3873/ad38d9>`_ and
+`Brandt (2024) <https://iopscience.iop.org/article/10.1088/1538-3873/ad38da>`_.
+The Poisson and read noise  computations are based on equations (27) and (28), defining
+:math:`\alpha_i`, the diagonal of :math:`C`, and :math:`\beta_i`, the off diagonal.
+
+This algorithm runs ramp fitting twice.  The first run allows for a first
+approximation for the slope, hence :math:`C`, as well as to take care for any jumps
```

Comment:
The computation is an iterative process.  An initial count rate guess (slope) is needed to compute the covariance matrix.

Also, I purposefully wrote "take care for" because the computations are done taking jumps into consideration.